css/components/portfolio.css

```diff
@@ -0,0 +1,95 @@
+.row2 {
```

Comment:
Shall we start.. :) 
Mano siulymas istrint viska nuo 1 eilutes iki 44.

**Kodel?:** 
Kaip supratau, ideja buvo, kad ta galerija labiau per viduriuka butu ir mazesnio dydzio? 
Galima atsikratyt tuo row2 visiskai (ir is index.html) ir geriau virsuj irasyt 
```
.container .row.portfolio .container-portfolio.col-12 { 
    grid-column: 2 / span 10;
 }
```
Sitaip mes permusim visiskai defaultini `.row` klases stiliu ir pritaikysim savo portfolio sekcija kaip mum patogiau. 

Tai reiktu index.html savo sekcijos row klasej pridet dar viena klase **portfolio** `<div class="row portfolio">`
ir prie container-portfolio klases pridet col-12 klase. `<div class="container-portfolio col-12">`
<hr>

Ir to viso `.row-title` kodo nereikia. Gaus defaultini
<hr>

O gal ideja viso sito buvo visai kitokia? 🤔

Reply:
Ideja buvo tokia, bet tavo pasiulyma neisprende mano problemos, arba as kazko nesuprantu ..

Reply:
![pakeitimai](https://user-images.githubusercontent.com/62422202/102007771-87bf7f80-3d34-11eb-96a6-5eac6592b13a.png)

Reply:
bet labai geri pastebejimai! Aciu

Reply:
@gvisis issprende! as ziopla.. miss type 🙃